src/pages/mesh/best-practices/performance.md

```diff
@@ -14,6 +14,8 @@ keywords:
 
 When performance testing edge meshes on API Mesh for Adobe Developer App Builder, you need to account for cold starts to get an accurate measurement of the performance.
 
+For performance testing, your edge mesh must be in a "Production" workspace. If your mesh existed before the September 23, 2024 release, you must run the `aio api-mesh update` command on your Production edge mesh before you can benefit from this enhancement.
```

Comment:
@revanth0212 is it only for "Production" workspaces in Dev Console or Production & Stage? 

@jhadobe we should specify production workspace in Developer Console as the term production has N different meanings for developers

Reply:
Yes @naomirobichaud this release is only for workspaces with the name `Production` in Dev Console.